### DISCUSSION TITLE
Can I update to an specific version in a standalone/distributed install?

### DISCUSSION THREAD

**Discussion by Carlos-mb** [Security-Onion-Solutions/securityonion#13589] [General] [answered]
Hi

I thought it would be great to be able to update/install a specific version, other than the latest one, so that you can choose the one you consider more stable.

Using soup -f you can decide the ISO, my question is, will it work on a standalone/distributed installation or will there be some kind of conflict during the update process because the ISO is different from the latest version?

Thanks

**Comment by reyesj2** [comment, marked as answer]
In an airgap deployment when running soup you are asked for the ISO location. Then soup will use that ISO for the update. So in that case you would be able to update to whatever version you wanted vs running non-airgap soup and updating to the latest version.
